README.md

```diff
@@ -9,16 +9,25 @@
 
 # The Open Source Observability Distribution
 
-Features that are not guarded behind a paywall.
-Opstrace installs secure, horizontally-scalable, open source observability in your cloud account combining open APIs with reduced toil.
-Opstrace is:
+Opstrace is a secure, horizontally-scalable, open source observability platform installed in your cloud account.
+It combines open APIs with reduced toil.
+
+Highlights:
 
 * Horizontally scalable for ingestion and queries.
 * Durable with inexpensive long-term retention.
-* Rigoriously tested end-to-end, freqently, for confident upgrades.
+* Rigoriously tested end-to-end.
+* Large efforts towards confident upgrades.
```

Comment:
I find it important to decouple the points of rigorous end-to-end testing, and the effort towards seamless upgrades.